Library/Formula/qt.rb

```diff
@@ -25,6 +25,9 @@ class Qt < Formula
     sha256 "c8a0fa819c8012a7cb70e902abb7133fc05235881ce230235d93719c47650c4e"
   end
 
+  # Make `HOMEBREW_PREFIX/lib/qt4/plugins` an additional plugin search path.
+  patch :DATA
```

Comment:
Qt 4.8 is closed for non-security submissions, if I'm not misreading http://lists.qt-project.org/pipermail/development/2015-August/022911.html …